py/trtorch/csrc/tensorrt_classes.h

```diff
@@ -17,6 +17,16 @@ namespace pyapi {
     return field_name;                      \
   }
 
+// TODO: Make this error message more informative
```

Comment:
Are we planning to address this later?

Reply:
Yeah, will need to think about it a bit more but shouldnt block this PR